planex/blobs.py

```diff
@@ -118,7 +118,8 @@ class GitBlob(Blob):
     """
 
     # pylint: disable=too-many-arguments
-    def __init__(self, spec, url, defined_by, prefix, commitish):
+    def __init__(self, spec, url, defined_by, spec_path, prefix, commitish):
+        self.spec_path = spec_path
```

Comment:
I assume that `spec_path` here means the path of the Source URL from the `.spec`, not the path of the `.spec` itself, and indeed when called below it gets a source path passed in. For clarity this could be called source_path or spec_source_path (to avoid confusion with the `spec` variable).

Reply:
It's the spec version of this path pre-override.